app/controllers/dois_controller.rb

```diff
@@ -689,14 +691,23 @@ def safe_params
     read_attrs_keys.each do |attr|
       p.merge!(attr.to_s.underscore => p[attr] || meta[attr.to_s.underscore] || p[attr]) if p.has_key?(attr) || meta.has_key?(attr.to_s.underscore)
     end
+
+    # handle alternate_identifiers
+    p[:identifiers] = Array.wrap(p[:alternate_identifiers]).map do |a|
+      { "identifierType" => a["alternateIdentifierType"], "identifier" => a["alternateIdentifier"] }
```

Comment:
Doesn't **bolognese** also need to be adjusted for this change ?? https://github.com/datacite/bolognese/blob/faddefb6edebdcd4060fd75e67c26176e13f5706/lib/bolognese/datacite_utils.rb#L124

Reply:
Please explain.

Reply:
well this is mostly a question. Since bolognse makes metadata transformation and used in various places. I was wondering whether this change will impact it.